### DISCUSSION TITLE
How to install Tailwind in multiple folders?

### DISCUSSION THREAD

**Discussion by Maykiyel** [tailwindlabs/tailwindcss#13277] [Help]
I'm making a project where I have an index.html file in the main folder with Tailwind installed and is working fine but inside my project is another folder(Apps) containing HTML pages even tho I already set the path in the config file tailwind still doesn't work.
![image_2024-03-19_193055395](https://github.com/tailwindlabs/tailwindcss/assets/156945500/daf48a94-db9a-40dd-b457-a46993dc0a17)
![image_2024-03-19_193117320](https://github.com/tailwindlabs/tailwindcss/assets/156945500/12164e2f-1442-4862-8490-b7f2464fa083)

**Comment by Vijaykv5** [comment]
Hi @ChaMi111111 
Please ensure that you've provided correct path of files (where you're using tailwind css) to be mentioned inside `tailwind.config.js` like this. 

```
/** @type {import('tailwindcss').Config} */
module.exports = {
  content: [
  //add path to files here
  ],
  // ...
}
```
If it didn't work, another way for easy fix is using a tailwind css cdn

**Reply by Maykiyel** [reply]
Yep, I already specified the path of the files inside tailwind.config.js. However, it still doesn't work. I'm trying to figure it out because I kind of don't feel good using cdn. If I do, will that affect my site's performance? If not then I'll just use cdn. Thanks!

**Reply by Vijaykv5** [reply]
@ChaMi111111 Is this working when you're just using your base level index file?
Just making sure that you configuration for tailwind is done correctly.

**Reply by Maykiyel** [reply]
@Vijaykv5  Yes it's working fine on my base index.html. I think I might have actually figured it out. I just have to either link the main.js module or the style.css inside the html page from the Apps folder. I'm not sure if that's the best way to do it though. I'm very new to tailwind and I'm not really familiar with its best practices
